app/models/post.rb

```diff
@@ -0,0 +1,18 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+  validates :name, presence: true
+  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  after_save :update_post_counter
```

Comment:
- [ ] In the post model you should have a CommentsCounter that must be an integer greater than or equal zero.
- [ ] In the post model you should have a LikesCounter must be an integer greater than or equal zero.

Reply:
Added validation for comments and like counters